server/api.js

```diff
@@ -27,8 +24,27 @@ router.get("/image", (req, res) => {
 	}
 });
 
-router.put("/image", (req, res) => {
-	res.json({ message: "Put image" });
-});
+router.put(
+	"/image",
+	uploadImage.single("image"), // our uploadImage middleware
+	(req, res) => {
+		/*
+           req.file = {
+             fieldname, originalname,
+             mimetype, size, bucket, key, location
+           }
+        */
+		console.log("request", req.file);
```

Comment:
We should remove console log statements from code we push. If we need logs, we can use the `logger` feature